### PR TITLE
Adding LabsMobile

### PR DESCRIPTION
**Pull request by borislalov**
I added LabsMobile as SMS provider. LabsMobile has good tariffs and a reliable service with direct routes for all carriers in European countries and good coverage in many other countries.

### CODE REVIEW

File: src/SimpleSoftwareIO/SMS/Drivers/LabsMobileSMS.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            $this->buildBody($data);
+
+            $this->getRequest();
+        }
+    }
+
+    /**
+     * Creates many IncomingMessage objects and sets all of the properties.
+     *
+     * @throws \RuntimeException
```
Comment (SimplyCorey) [inline src/SimpleSoftwareIO/SMS/Drivers/LabsMobileSMS.php:69]: Please add the params to the docblock.

File: src/SimpleSoftwareIO/SMS/Drivers/LabsMobileSMS.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+     */
+    protected function processReceive($rawMessage)
+    {
+        throw new \RuntimeException('LabsMobile does not support Inbound API Calls.');
+    }
+
+    /**
+     * Checks the server for messages and returns their results.
+     *
+     * @throws \RuntimeException
```
Comment (SimplyCorey) [inline src/SimpleSoftwareIO/SMS/Drivers/LabsMobileSMS.php:79]: Please add the params to the docblock.

File: src/SimpleSoftwareIO/SMS/Drivers/LabsMobileSMS.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+     */
+    public function checkMessages(Array $options = array())
+    {
+        throw new \RuntimeException('LabsMobile does not support Inbound API Calls.');
+    }
+
+    /**
+     * Gets a single message by it's ID.
+     *
+     * @throws \RuntimeException
```
Comment (SimplyCorey) [inline src/SimpleSoftwareIO/SMS/Drivers/LabsMobileSMS.php:89]: Please add the params to the docblock.

File: src/SimpleSoftwareIO/SMS/SMSServiceProvider.php [hunk 4]
```diff
@@ -90,6 +174,7 @@ private function setSMSDependencies($sms, $app)
      */
     public function provides()
     {
-        return array('sms', 'sms.sender');
+        return array('sms', 'emailsms', 'labsmobilesms', 'twiliosms', 'mozeosms', 'eztextingsms', 'callfiresms');
```
Comment (SimplyCorey) [inline src/SimpleSoftwareIO/SMS/SMSServiceProvider.php:177]: Please revert this change.

File: src/config/sms.php [hunk 5]
```diff
@@ -50,8 +50,8 @@
 */
 
 return [
-    'driver' => 'Selected Driver',
-    'from' => 'Your Number or Email',
```
Comment (SimplyCorey) [inline src/config/sms.php:54]: Please revert these changes as well.

File: src/SimpleSoftwareIO/SMS/SMSServiceProvider.php [hunk 6]
```diff
@@ -60,27 +68,103 @@ public function register()
...
-        $sms->setQueue($app['queue']);
+        $provider = new CallFireSMS(new Client);
+
+        $provider->setUser(config('sms.callfire.app_login'));
+        $provider->setPassword(config('sms.callfire.app_password'));
+
+        return $provider;
+    }
+
+    protected function buildLabsMobile()
```
Comment (SimplyCorey) [inline src/SimpleSoftwareIO/SMS/SMSServiceProvider.php:139]: A change was made to the latest master and moved this logic to the DriverManager.  Please update your pull request.

**Comment by SimplyCorey** [comment]
Thank you for your addition!  Please perform the requested changes and Ill glad merge the pull request.

**Comment by borislalov** [comment]
Changes made.

**Comment by borislalov** [comment]
I've made the changes you requested, con you merge the pull request? Thanks.

**Comment by SimplyThomas** [comment]
One last look @SimplyCorey ?

**Comment by SimplyCorey** [comment]
@borislalov Thank you for your hard work!  I am unable to merge this because of merge conflicts.  I fixed them in #17.
